### ISSUE TITLE
Postgres does not run scripts against target db

### ISSUE THREAD

**Issue by worldspawn** [chucknorris/roundhouse#207]
Using a command like: `.\packages\roundhouse.0.8.6\bin\rh.exe -d=calcengine --dt=postgres -f=".\database\" -c="host=localhost;user id=postgres;password=password" --noninteractive` results in the rather useless outcome of a database called `calcengine` being created and then all scripts are executed against the default database (`postgres`). 

If i include the database name in the connection string it fails to connect because the database does not exist (when it does not exist).

**Comment by lukemcgregor** [comment]
not 100% sure but i dont think you can use both -c and -d, try putting you database into the connection string eg:

    -c="host=localhost;database=calcengine;user id=postgres;password=password"

**Comment by worldspawn** [comment]
I checked the source and in the absence of -d it tries to read it from the connection string. It seems the solution it is have the database name in the connection (-c) and you must provide an admin connection string without a database name in it. When creating a db it will use the admin connstr then switch to the normal which should now work as the db exists.

Pretty minor issue (when you know the solution) but I did not have to do that when targeting MSSQL.
